mchf-eclipse/drivers/ui/radio_management.c

```diff
@@ -1133,6 +1133,18 @@ bool RadioManagement_IsApplicableDemodMode(uint32_t demod_mode)
     case DEMOD_SAM:
         retval =( ts.flags1 & FLAGS1_SAM_ENABLE) != 0;        // is SAM enabled?
         break;
+    case DEMOD_SSBSTEREO:
+    case DEMOD_SAMSTEREO:
+    case DEMOD_IQ:
+    	if(!ts.stereo_enable)
+    	{
+    		retval = false;
+    	}
+    	else
+    	{
+    		retval = true;
+    	}
+    	break;
```

Comment:
retval = ts.stereo_enable would do the same job in a single line ;-)

Reply:
Oh, thanks! Acknowledged ;-). Should I change it or will the compiler do the job of simplifying?